internal/broker/bind_create_test.go

```diff
@@ -272,6 +275,58 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		}`, fixture.PlanId, customExpirationSeconds), t)
 		require.Equal(t, http.StatusBadRequest, response.StatusCode)
 	})
+
+	t.Run("should return 404 for not existing binding", func(t *testing.T) {
```

Comment:
Are there test cases testing scenarios with many bindings for the same `service instance`?

Reply:
But we need to test scenario where there is many bindings per instance and we query for particular binding